compiler/frontend/beam/src/main/java/org/apache/nemo/compiler/frontend/beam/PipelineTranslator.java

```diff
@@ -198,11 +196,29 @@ private static void parDoMultiOutputTranslator(final TranslationContext ctx,
         pValueWithTupleTag.getKey()));
   }
 
+  private static GroupByKeyTransform createGBKTransform(
+    final TranslationContext ctx,
+    final TransformVertex transformVertex) {
+    final AppliedPTransform pTransform = transformVertex.getNode().toAppliedPTransform(PIPELINE.get());
+    final PCollection<?> mainInput = (PCollection<?>)
+      Iterables.getOnlyElement(TransformInputs.nonAdditionalInputs(pTransform));
+    final TupleTag mainOutputTag = new TupleTag<>("main output");
+
+    return new GroupByKeyTransform(
+      getOutputCoders(pTransform),
+      mainOutputTag,
+      Collections.emptyList(),
+      mainInput.getWindowingStrategy(),
+      Collections.emptyList(), /* side inputs */
```

Comment:
Briefly comment that GBK does not have side inputs?

Reply:
Done